example/swagger-files/petstore.json

```diff
@@ -229,6 +229,13 @@
           "description": "Updated status of the pet",
           "required": false,
           "type": "string"
+        }, {
```

Comment:
Instead of adding this here, can you add it to this file like these ones: https://github.com/readmeio/api-explorer/blob/master/example/swagger-files/types.json#L40-L60